packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -406,6 +435,12 @@ export class LocalIntervalCollection<TInterval extends ISerializableInterval> {
         }
     }
 
+    public findInterval(start: number, end: number) {
```

Comment:
does this return an array, or a single interval?

Reply:
this method is odd. which interval does it return? with or without id? or either?

Reply:
It returns a single interval, basically, the first one it finds that matches (start, end). Yeah, maybe that's odd, and it would be nicer to return an array. Then, for the delete(start, end) case, the caller could walk the array doing the individual deletes.